addons/mail/static/src/components/composer/composer.xml

```diff
@@ -2,167 +2,165 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ComposerView" owl="1">
-        <t t-if="composerView">
```

Comment:
remove `</t>`

Reply:
Looks properly removed, please note there's a local `t-name` in the file, so this `t-name="mail.ComposerView"` ends at 2/3rd of the file